### PR TITLE
Add Pixlr presence

### PR DESCRIPTION
**Pull request by Uhev-png**
Here's proof that my presence works

<a href="https://imgur.com/UOX7EPs"></a>

<a href="https://imgur.com/zNs1uC9"></a>

### CODE REVIEW

File: websites/P/Pixlr/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+    "author": {
+      "name": "Uhevela",
+    "id": "616969786718683167"
+    },
+    "url": "pixlr.com",
+    "description": {
+    "en": "Pixlr is a cloud-based set of image tools and utilities, including a number of photo editors, a screen grabber browser extension, and a photo sharing service. The suite was intended for non-professionals, however the apps range from simple to advanced photo editing."
+    },
+    "service": "pixlr",
```
Comment (Bas950) [inline websites/P/Pixlr/dist/metadata.json:10]: service name and folder name must be the same (cap letters included)

File: websites/P/Pixlr/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    "version": "1.0.0",
+    "logo": "https://i.imgur.com/LJyc7pb.jpg",
+    "thumbnail": "https://i.imgur.com/9OWj9UN.png",
+    "color": "#383838",
+    "tags": [
+    "editor",
+    "pics"
+    ],
+    "category": "Img editor"
```
Comment (Bas950) [inline websites/P/Pixlr/dist/metadata.json:19]: this is not a valid category

File: websites/P/Pixlr/presence.ts [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+      
+      presence.setActivity(presenceData);
+    } else if (document.location.pathname == "/x/") {
+      presenceData.details = "Edits an image",
+      presenceData.state = "Playful editor";
+
+      presence.setActivity(presenceData);
+    } else if (document.location.pathname == "/pricing/") {
+      presenceData.details = "Looking price";
+
```
Comment (Bas950) [inline websites/P/Pixlr/presence.ts:30]: setActivity seems to be missing

File: websites/P/Pixlr/presence.ts [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    } else {
+      const presenceData: PresenceData = {
+        largeImageKey: "pixlr"
+      };
```
Comment (Bas950) [inline websites/P/Pixlr/presence.ts:34]: shouldn't be needed again here.

File: websites/P/Pixlr/presence.ts [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+      presence.setActivity(presenceData);
+    } else if (document.location.pathname == "/x/") {
+      presenceData.details = "Edits an image",
+      presenceData.state = "Playful editor";
+
+      presence.setActivity(presenceData);
+    } else if (document.location.pathname == "/pricing/") {
+      presenceData.details = "Looking price",
+      presenceData.state = "Shop";
+    }
```
Comment (Bas950) [inline websites/P/Pixlr/presence.ts:31]: still missing set activity

File: websites/P/Pixlr/presence.ts [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+      presence.setActivity(presenceData);
```
Comment (Bas950) [inline websites/P/Pixlr/presence.ts:32]: shouldnt that set activity be in your if statement
after you have done that upload new proof of it working

**Comment by Alanexei** [comment]
This pull request will be closed in 2 days.

**Comment by frozaum** [comment]
1 day

**Comment by Bas950** [comment]
Hasn't made required changes within a week. - Closed

**Pull request closed without merging**